lib/epochs.js

```diff
@@ -509,24 +511,46 @@ function epochNodeStream(ssb, groupId, opts = {}) {
 
   return deferredSource
 }
+function getRepeat(ssb, key, cb) {
+  ssb.db.get(key, (err, rootVal) => {
+    if (err) {
+      setTimeout(() => {
+        return getRepeat(ssb, key, cb)
+      }, 100)
+    } else {
+      return cb(null, rootVal)
+    }
+  })
+}
 function getGroupInit(ssb, groupId, cb) {
-  ssb.box2.getGroupInfo(groupId, (err, info) => {
-    // prettier-ignore
-    if (err) return cb(clarify(err, 'Failed to get group info for ' + groupId))
-    if (!info) return cb(new Error('Unknown group'))
-
-    // Fetch the tangle root
-    ssb.db.get(info.root, (err, rootVal) => {
-      // prettier-ignore
-      if (err) return cb(clarify(err, 'Failed to load group root with id ' + info.root))
-
-      if (!isInitRoot(rootVal))
-        // prettier-ignore
-        return cb(clarify(new Error(isInitRoot.string), 'Malformed group/init root message'))
-
-      cb(null, { key: info.root, value: rootVal })
+  pull(
+    ssb.box2.getGroupInfoUpdates(groupId),
+    pull.take(1),
+    pull.drain((info) => {
```

Comment:
I suspect this change might not have been needed, but idk if it hurts either?

Reply:
:fire: needs to be able to handle errors on drain

```js
pull.drain(
  (info) => { },
  (err) => { }
)
```